### DISCUSSION TITLE
What happen if we delete the record with unique field

### DISCUSSION THREAD

**Discussion by code-blooded-guy** [Sunbird-RC/community#329] [General]
We have one field Enrollment Number in the DB which is **unique**.
we have added one record with a value of that field = 99999.
I am deleting this record from RC.
I am adding the new record with value 99999.

Do you know if the record will be created or shows the error in RC?

**Comment by pramodkvarma** [comment]
Did you try? It should allow insert again given the earlier record is deleted. But, if you soft delete (like revocation of a credential record), then it should fail.
